README.md

```diff
@@ -1,2 +1,52 @@
-# mpi-operator
-Repository for the MPI operator.
+# MPI Operator
+
+The MPI Operator makes it easy to run allreduce-style distributed training.
+
+## Build
+
+Check out the code:
+```shell
+mkdir -p ${GOPATH}/src/github.com/kubeflow
+cd ${GOPATH}/src/github.com/kubeflow
+git clone https://github.com/kubeflow/mpi-operator.git
+cd mpi-operator
+```
+
+Build and push the `mpi-operator` Docker image:
+```shell
+docker built -t rongou/mpi-operator:0.1.0 -f cmd/mpi-operator/Dockerfile .
```

Comment:
I think we could use kubeflow/image-name insead of rougou for better maintenance.

Reply:
Do you know who owns kubeflow on docker hub? I don't have access to it. Tracking at https://github.com/kubeflow/mpi-operator/issues/4.